crates/biome_service/src/file_handlers/css.rs

```diff
@@ -350,51 +352,52 @@ fn lint(params: LintParams) -> LintResults {
                 .count();
 
             info!("Analyze file {}", params.path.display());
-            let (_, analyze_diagnostics) = analyze(&tree, filter, &analyzer_options, |signal| {
-                if let Some(mut diagnostic) = signal.diagnostic() {
-                    // Do not report unused suppression comment diagnostics if this is a syntax-only analyzer pass
-                    if ignores_suppression_comment
-                        && diagnostic.category() == Some(category!("suppressions/unused"))
-                    {
-                        return ControlFlow::<Never>::Continue(());
-                    }
+            let (_, analyze_diagnostics) =
+                analyze(&tree, filter, &analyzer_options, Vec::new(), |signal| {
```

Comment:
This change made me realize that plugin loading is still missing from the workspace, so although everything else is in place, the invocations to the analyzers still always pass an empty vector for the plugins. The same is true for the JS analyzer currently.

I'll address this in a separate PR.